Index: topshelf/autocomplete_light_registry.py
===================================================================
--- topshelf/autocomplete_light_registry.py	(revision )
+++ topshelf/autocomplete_light_registry.py	(revision )
@@ -1,13 +0,0 @@
-__author__ = 'zhilabug'
-
-import autocomplete_light
-from topshelf.models import UserIngred
-
-# This will generate a PersonAutocomplete class
-autocomplete_light.register(UserIngred,
-    # Just like in ModelAdmin.search_fields
-    search_fields=['^ing_master'],
-    # This will actually html attribute data-placeholder which will set
-    # javascript attribute widget.autocomplete.placeholder.
-    autocomplete_js_attributes={'placeholder': 'Type an ingredient',},
-)
\ No newline at end of file
Index: topshelf/static/views/pantry.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- topshelf/static/views/pantry.html	(revision )
+++ topshelf/static/views/pantry.html	(revision )
@@ -1,9 +1,9 @@
  <div class='container-fluid'>
+    <div> Need to add a list of inventory items, that can be deleted with a click. </div>
     <form ng-submit="submitForm()" >
         <h4>Stock Your Pantry</h4>
-        <input type="text" ng-model="pantry.ing_master" typeahead="item.resource_uri as item.ing for item in list_items | filter:$viewValue | limitTo:30" class="form-control">
+        Add an ingredient: <input type="text" ng-model="pantry.ing_master" typeahead="item.resource_uri as item.ing for item in list_items | filter:$viewValue | limitTo:30" class="form-control">
         <br>
-        <div> Need to add a list of inventory items, that can be deleted with a click. </div>
         <div>
         Purchase date: <input type="text" ng-model="pantry.purch_date">
         </div>
\ No newline at end of file
Index: topshelf/admin.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- topshelf/admin.py	(revision )
+++ topshelf/admin.py	(revision )
@@ -1,10 +1,6 @@
 from django.contrib import admin
 from topshelf.models import IngredMaster, UserIngred
-# import autocomplete_light
 
-# class UserIngredAdmin(admin.ModelAdmin):
-#     # This will generate a ModelForm
-#     form = autocomplete_light.modelform_factory(UserIngred)
 
 admin.site.register(UserIngred)
 admin.site.register(IngredMaster)
Index: topshelf/forms.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- topshelf/forms.py	(revision )
+++ topshelf/forms.py	(revision )
@@ -1,4 +1,3 @@
-import autocomplete_light
 from django.forms import ModelForm, DateField
 from topshelf.models import IngredMaster, UserIngred
 
@@ -48,5 +47,4 @@
 #     select3 = forms.ChoiceField(UserIngred.ing_master)
 #     diet_choice = forms.ChoiceField(DietMaster.diet)
 
-# WidgetForm = autocomplete_light.modelform_factory(UserIngred)
 #Add date widget?
\ No newline at end of file
Index: topshelf/views.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- topshelf/views.py	(revision )
+++ topshelf/views.py	(revision )
@@ -13,9 +13,6 @@
 from django.shortcuts import render, redirect
 
 from topshelf.forms import SignupForm, LoginForm
-
-
-
 from topshelf.models import UserIngred
 
 
